app/src/main/res/layout/widget_media_controls.xml

```diff
@@ -57,58 +57,82 @@
                 android:layout_alignParentTop="true"
                 android:layout_alignParentEnd="true"
                 android:padding="8dp"
-                android:visibility="invisible"
                 android:tint="?colorWidgetOnBackground"
                 android:src="@drawable/app_icon_round" />
 
+            <TextView
```

Comment:
Please set the text color like on the other `TextView`s to make sure the text is readable and fits in when using the dynamic color theme:

```
android:textColor="?colorWidgetOnBackground"
```